src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -8,66 +8,72 @@ public class Exercise extends ExerciseBase {
 
 
     // 1. Change the value of the member below to be the result of adding numOne and numTwo together
-    public int numOnePlusTwo = 0;
+    public int numOnePlusTwo = numOne + numTwo;
+
 
 
     // 2. Change the value of the member below to be the result of multiplying numThree by numTwo
-    public int numThreeTimesNumTwo = 0;
+    public int numThreeTimesNumTwo = numThree * numTwo;
 
 
     // 3. Change the value of the member below to be the result of dividing numThree by numOne
-    public int numThreeDividedByNumOne = 0;
+    public int numThreeDividedByNumOne = numThree / numOne;
 
 
     // 4. Change the value of the member below to be the result of subtracting numOne from numThree
-    public int numThreeMinusNumOne = 0;
+    public int numThreeMinusNumOne = numThree - numOne;
 
 
     // 5. Change the value of the member below to be the sum of numOne, numTwo and numThree
-    public int sum = 0;
+    public int sum = numOne + numTwo + numThree;
 
 
     // 6. Change the value of the member below to be the sum of numOne, numTwo and numThree divided by numOne
-    public int numBytes = 0;
+    public int numBytes =  sum / numOne ;
 
 
     // 7. Create a public char member named lastLetter containing the last letter of the English alphabet
-
+    public char lastLetter = 'Z';
 
 
     // 8. Create a public float member named pi that contains the value of pi to two decimal places
 
-
+    public float pi = 3.14f;
 
     // 9. Create a public double member named piD that contains the value of pi to 5 decimal places
 
-
+    public double piD = 3.14159;
 
     public String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
     public String firstName = "Jane";
     public String lastName = "Smith";
 
-    // 10. Create a public member named fullName that contains the value of firstName and lastName concatenated together with a space in between
+        // 10. Create a public member named fullName that contains the value of firstName and lastName concatenated together with a space in between
 
+    public String fullName = firstName + " " + lastName;
 
 
     // 11. Create a public char member named tenthLetter that contains the tenth letter in the alphabet member above.
     // If you need help, here's some documentation: https://www.w3schools.com/java/ref_string_charat.asp
 
+    public char tenthLetter = 'J';
```

Comment:
Technically correct, but we were looking for `alphabet.charAt(9);` (directly referencing the alphabet member.

Reply:
My bad, noted.